DATA/common/setenv.sh

```diff
@@ -169,9 +169,6 @@ ITSMFT_STROBES=""
 [[ ! -z ${ITS_STROBE:-} ]] && ITSMFT_STROBES+="ITSAlpideParam.roFrameLengthInBC=$ITS_STROBE;"
 [[ ! -z ${MFT_STROBE:-} ]] && ITSMFT_STROBES+="MFTAlpideParam.roFrameLengthInBC=$MFT_STROBE;"
 
-MFTMCH_NCANDIDATES_OPT=
```

Comment:
@aferrero2707 : Sorry, but this line must stay in setenv.sh, since this one is also used in online :(.

Reply:
@davidrohr why you say this is needed for online as well? AFAIK we are not using the option for multiple candidates in online.

In the previous PR I have put the lines in `setenv.sh` because I was not aware of the `setenv_extra.sh` specific for async processing...

Reply:
Ah, ok, I didn't know that.
I thought also MFTMCH_NCANDIDATES_OPT must be set for sync, since it is used.
But it is actually used like `[[ ! -z ${MFTMCH_NCANDIDATES_OPT:-} ]] &&` , so ok if unset.
I'll revert my moving of the fail. Thx for checking and sorry for the confusion.